docs/develop/01_sdk/02_cookbook/02_web3names/03_release.md

```diff
@@ -10,14 +10,39 @@ import ReclaimDeposit from '!!raw-loader!@site/code_examples/sdk_examples/src/co
 
 If a web3name is no longer needed, either the DID owner or the deposit payer can release it, with deposit being released and returned to the original payer.
 
+## Releasing a Web3name by the DID Owner
+
 In the case of the DID owner willing to release the web3name, the following snippet provides a reference implementation on how to achieve that.
 
 <TsJsBlock>
   {Release}
 </TsJsBlock>
 
+In the code above, the releaseWeb3Name function takes the following parameters:
+
+* **did**: The DID URI of the owner.
+* **submitterAccount**: The keyring pair of the submitter.
+* **signCallback**: The sign extrinsic callback function. This function is used to sign the extrinsic, read more that in [the SignCallback section](../07_signCallback.md).
+
+The function `releaseWeb3Name` uses the KILT SDK to create a *web3name release transaction* using `api.tx.web3Names.releaseByOwner`.
```

Comment:
I think adding comments to the code make it less readable. e.g. hard to spot what is actually going on, finding the function calls in between all the comments.
Yes this can get outdated, but I would hope that not only the code example is checked when there is an error, but also the guide where the code is used. 😓